samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -142,6 +153,16 @@ Vendor specific commands can be divided into different categories as follows:
   The two most significant bits are calculated by the DTM module.
   This is possible because the 6 least significant bits of all valid TX power values are unique.
   The TX power can be modified only when no Transmitter Test or Receiver Test is running.
+* If the Length field is set to ``3``(symbol ``NRF21540_ANTENNA_SELECT``), the Frequency field sets the nRF21540 FEM antenna.
+  The valid values are:
+
+     * 0 - ANT1 enabled, ANT2 disabled
+     * 1 - ANT1 disabled, ANT2 enabled
+
+* If the Length field is set to ``4`` (symbol ``NRF21540_GAIN_SET``), the Frequency field sets the nRF21540 FEM TX gain value in arbitrary units.
+  The valid gain values are specified in the nRF21540 product-specific range from 0 to 31.
+* If the Length field is set to ``5`` (symbol ``NRF21540_ACTIVE_DELAY_SET``), the Frequency field sets the nRF21540 FEM activation delay in microseconds relative to a radio start.
+  By default, this value is set to (radio ramp-up time - nRF21540 TX/RX settling time).
```

Comment:
Proposing: By default, this value is set to radio ramp-up time (nRF21540 TX/RX settling time).

Reply:
I mean there radio ramp-up time subtract nRF21540 TX/RX settling time.

Reply:
Oh, I see. My misunderstanding.